bitnami/postgresql/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   category: Database
 apiVersion: v2
-appVersion: 11.12.0
+appVersion: 11.13.0
```

Comment:
Sorry, just one thing. This is not the version that you need to change it. Please leave the version to 11.12.0. You need to modify the version version in the end of the file. This is the reason that you are having linter errors

Reply:
Good to know, thanks! I reverted the `appVersion` and changed the `version` accordingly.